loop.rb

```diff
@@ -1,5 +1,6 @@
 # assumes function deployed at least once
 puts `fn routes update myapp /fdk-ruby --format default`
+sleep 5
```

Comment:
Why the sleep?

Reply:
Delay to allow format switch to complete (see below).  Will also shorten to 1 second.